src/browser/modules/EditorInfo/utils.js

```diff
@@ -34,7 +34,14 @@ export const getStringValue = value => {
     return value ? 'true' : 'false'
   }
 
-  // FIXME handle if value is an array
+  if (typeof value === 'object') {
```

Comment:
i need to see it in working.
let me know when we can.

Reply:
use Array.isArray(value) to check if value is an array or not

Reply:
Its done